src/EA.Weee.Web/Views/Shared/Producer/ViewOrganisation/TotalEEEDetails.cshtml

```diff
@@ -43,6 +43,12 @@
                             </td>
                         </tr>
                     }
+                    <tr class="govuk-table__row">
+                        <th scope="col" class="govuk-table__header govuk-!-text-align-left" style="width: 28%;">Selling technique(s) that you use</th>
```

Comment:
@will-hope98  assume this just works for both internal and external?

Reply:
@JohnPadgettCivica ah no, just checked internal and it's broken it

Reply:
Fixed now